main.js

```diff
@@ -0,0 +1,101 @@
+// creates a div element to hold the Typing Tutor Text
+var $title = document.createElement('div')
+$title.textContent = "Typing Tutor"
+document.body.appendChild($title)
+$title.style.fontSize = '50px'
```

Comment:
Define a CSS class in your `styles.css` and add it to the `$title.classList` instead of monkeying with `$title.style`. We only do that is special scenarios.

Reply:
OK will do.

Reply:
Done. See previous comment re: two new classes created.